### ISSUE TITLE
main's vulkan-samples.exe does not work on Windows

### ISSUE THREAD

**Issue by VladyslavSan** [KhronosGroup/Vulkan-Samples#786]
So basically cloning and building samples + providing correct cmd arguments for it to run sample result in error:
```
[info] Logger initialized
[info] Initializing Vulkan sample
[info] Vulkan debug utils enabled (VK_EXT_debug_utils)
[info] Extension VK_EXT_debug_utils found, enabling it
[info] VK_EXT_validation_features is available, enabling it
[info] Extension VK_KHR_get_physical_device_properties2 found, enabling it
[info] Extension VK_KHR_win32_surface found, enabling it
[info] Enabled Validation Layers:
[error] [framework\platform\platform.cpp:169] Error Message: Could not create Vulkan instance : ERROR_EXTENSION_NOT_PRESENT
[error] [framework\platform\platform.cpp:170] Failed when running application descriptor_buffer_basic
Press return to continue
```

The commit ID is 58eb923ba4cdb564088d1f6b33482c774bdc9bfa.
It's pretty strange that it doesn't work out of the box on Windows 11, RTX 4080, and latest drivers. Btw, my own app works.

Clicking here and there I've managed to make it work but it still feels like a bug. So if I open Vulkan Configurator, select "override layers" and select "Validation" then the app runs. But if I select "Layers Fully Controlled by the Vulkan Application" it doesn't work.

I had VulkanSDK of version 1.3.250.1 but updated to the new 1.3.261.0 just in case it fixes the problem but it doesn't.

**Comment by SaschaWillems** [comment]
We are already working on a fix, see #774

**Comment by SaschaWillems** [comment]
#774 has been merged, this should be fixed now. If not, feel free to open a new issue.
